src/Deserialization/deserialization.jl

```diff
@@ -82,6 +84,7 @@ TBEventFileIterator(fstream) = TBEventFileIterator(fstream, typemax(Int))
 function Base.iterate(it::TBEventFileIterator, state=0)
     eof(it.fstream) && return nothing
     ev=read_event(it.fstream)
```

Comment:
If it can return nothing
Then should we call this.
`try_read_event`?

Reply:
I'd rather put some 'try-check' logic in the iterate method than change the behaviour of `read_event`